src/availability_filter.py

```diff
@@ -0,0 +1,9 @@
+class AvailabilityFilter:
+    def __init__(self, subfilter, availability):
+        self.subfilter = subfilter
+        self.availability = availability
+
+    def is_match(self, beer):
+        if beer.is_available != self.availability:
```

Comment:
This should be done by quantity, not by adding a field to the beer object.

Reply:
I hadn't thought about allowing for True and False for `availability`, but this is a good idea.

Make sure there is a test for positive, zero *and* negative - I recall seeing negative quantities in the data.